addon/components/boolean-input.js

```diff
@@ -0,0 +1,46 @@
+import _ from 'lodash'
+import computed, {readOnly} from 'ember-computed-decorators'
+import Input from './input-base'
```

Comment:
InputBase would be good too, as it implies the abstract nature of the class and maps to the module name